frontend/src/components/kl-char.js

```diff
@@ -38,18 +36,6 @@ export default {
                         toastr.error(this.$t('errors.default'));
                 });
             }
-        },
-        critChanceControl: function () {
-            if (this.char.critChance < 0) this.char.critChance = 0;
-        },
-        critDmgControl: function () {
-            if (this.char.critDmg < 0) this.char.critDmg = 0;
-        },
-        dmgControl: function () {
-            if (this.char.dmg < 0) this.char.dmg = 0;
-        },
-        atkSpdControl: function () {
```

Comment:
ah, sure, didn't notice its about char and not about item.
what about letters btw?